CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Versions
 
+## v1.0.0-rc.1
+
+_Dec 11, 2025_
+
+### General Changes
+
+- **Breaking change:** Rename packages to use the @base-ui org.<br />
```

Comment:
I think this may reference the GitHub user. Let's use a code block to be safe

```suggestion
- **Breaking change:** Rename packages to use the `@base-ui` org.<br />
```